steely/config.py.sample

```diff
@@ -10,3 +10,7 @@ TM_API_PASS = ''
 ADMINS = ('', )
 COMMAND_PREFIX = '.'
 BREWERYDB_API_KEY = ''
+DEEP_API_KEY = ''
```

Comment:
huh?

Reply:
yeah really shouldnt be in this commit, but its from the deep plugin. that line as never added

Reply:
when we delete the bot and start over again we can add it to the config from the beginning, nbd